### PR TITLE
Add SBL examples for phytec lyra am62ax board

### PR DESCRIPTION
**Pull request by jma-phytec**
New sbl_uart_uniflash_multistage and sbl_null are added to support Phytec lyra am62ax board.  DDR and OSPI flash drivers are updated.

### CODE REVIEW

File: examples/drivers/boot/sbl_null/phyboard-lyra-am62xx/r5fss0-0_nortos/ti-arm-clang/makefile_ccs_bootimage_gen [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+SYSFW_LOAD_ADDR=0x40000
+BOARDCFG_LOAD_ADDR=0x67000
+BOARDCFG_SBLDATA_LOAD_ADDR=0x43c3c800
+BOARDCFG_BLOB=$(MCU_PLUS_SDK_PATH)/source/drivers/sciclient/sciclient_default_boardcfg/am62ax/boardcfg_blob.bin
+BOARDCFG_SBLDATA_BLOB=$(MCU_PLUS_SDK_PATH)/source/drivers/sciclient/sciclient_default_boardcfg/am62ax/boardcfg_sbldata_blob.bin
+ENABLE_SBLDATA=yes
+
+SBL_RUN_ADDRESS=0x43C00000
+
+SBL_PREBUILT_PATH=$(MCU_PLUS_SDK_PATH)/tools/boot/sbl_prebuilt/phyboard-lyra-am62xx
```
Comment (dnltz) [inline examples/drivers/boot/sbl_null/phyboard-lyra-am62xx/r5fss0-0_nortos/ti-arm-clang/makefile_ccs_bootimage_gen]:
```suggestion
SBL_PREBUILT_PATH=$(MCU_PLUS_SDK_PATH)/tools/boot/sbl_prebuilt/phyboard-lyra-am62xx
```
```suggestion
SBL_PREBUILT_PATH=$(MCU_PLUS_SDK_PATH)/tools/boot/sbl_prebuilt/phyboard-lyra-am62axx
```

**Review by dnltz — CHANGES_REQUESTED** [review]
Wrong baord name.

File: examples/drivers/boot/sbl_null/phyboard-lyra-am62xx/r5fss0-0_nortos/ti-arm-clang/makefile_ccs_bootimage_gen (hunk 1 above)
Comment (jma-phytec) [reply examples/drivers/boot/sbl_null/phyboard-lyra-am62xx/r5fss0-0_nortos/ti-arm-clang/makefile_ccs_bootimage_gen]: Fixed it in all sbl_null and sbl_uart_uniflash examples.  Thank you.

**Review by dnltz — APPROVED** [review]
@jma-phytec can you provide some documentation how to build and flash those examples?

**Comment by jma-phytec** [comment]
A page (https://wiki.phytec.com/pages/viewpage.action?pageId=863601243) was drafted to use these SBL projects.

**Review by dnltz — APPROVED** [review]

**Pull request merged**